sovrin_common/constants.py

```diff
@@ -1,6 +1,94 @@
 from typing import NamedTuple
 
+from plenum.common.constants import TXN_TYPE, TARGET_NYM, ORIGIN, DATA, RAW, ENC, HASH, NAME, VERSION, TYPE, \
+    POOL_TXN_TYPES, ALIAS, \
+    VERKEY
+from sovrin_common.roles import Roles
+from sovrin_common.transactions import SovrinTransactions
+
 Environment = NamedTuple("Environment", [
     ("poolLedger", str),
     ("domainLedger", str)
 ])
+
+ROLE = 'role'
```

Comment:
I agree with you. While doing my changes, I just processed a TODO to rename txn.py to constants.py, because previously it was even worse: we had a module with all constants, but the name of the module was confusing (txn.py contained not only constants related to transactions).
I will create a task to re-factor usage of constants in the code (because I think it's a re-factoring task which is not in the scope of the current story).
In addition to splitting the constants file, I believe we need more well-defined types and enumerations, not just string constants.